entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ ! -z "$WAIT_TIME" ]; then
```

Comment:
Maybe put a comment in the script to let people know why it exists, otherwise I'm guessing we'll end up with PRs to remove this :)